packages/client/src/comp/states/Minnesota.tsx

```diff
@@ -29,20 +22,25 @@ export const Minnesota = () => {
 
   return <SignatureBase<MinnesotaInfo>enrichValues={enrichValues}>
     <p>Minnesota requires voters to confirm their identify using one of the following types of identification</p>
-    <Select ref={idTypeRef} label='Identification Type' defaultValue='Select' {...{required: true}}>
-      <Option key={0} hidden={true}/>
-      {[...minnesotaIdentityType].sort().map((value, key) => {
-        return <Option value={value} key={key+1} label={value}/>
-      })}
-    </Select>
-    <p>Enter the relevant information based on your choice above.  If &apos;None&apos; 
+    <Select
+      label='Identification Type'
+      value={idType}
+      options={[...minnesotaIdentityType]}
+      onChange={v => setIdType(v)}
+      {...{required: true}}
```

Comment:
why not `required=true`?

Reply:
Now that I'm looking at it, I don't understand why we need `required` on `<Select/>` in the first place, I'm going to remove this prop from the others `<Select/>` as well since we'll always have values on them.

Reply:
+1